engines/scumm/gfx_gui.cpp

```diff
@@ -3130,7 +3613,43 @@ void ScummEngine::updateMainMenuControls() {
 	_system->updateScreen();
 }
 
+void ScummEngine::updateMainMenuControlsSegaCD() {
+	char msg[256];
+	int yConstant = _virtscr[kMainVirtScreen].topline + (_virtscr[kMainVirtScreen].h / 2);
+	bool isJap = _language == Common::JA_JPN;
+
+	if (_menuPage == GUI_PAGE_MAIN) {
+		strncpy(_mainMenuTextSpeedSlider, "\x3a\x3a\x3a\x3a\x3a\x3a\x3a\x3a\x3a\x3a", sizeof(_mainMenuTextSpeedSlider));
```

Comment:
Please describe the string in a comment. Something like `// Horizontal line`

Same for `\x3b` below